virtualenv_tools.py

```diff
@@ -167,6 +167,33 @@ def get_new_path(filename):
                 update_pyc(filename, local_path)
 
 
+def _update_pth_file(pth_filename, orig_path):
+    with open(pth_filename) as f:
+        lines = f.readlines()
+    changed = False
+    for i, line in enumerate(lines):
+        val = line.strip()
+        if val.startswith('import ') or not os.path.isabs(val):
+            continue
+        changed = True
+        relto_original = os.path.relpath(val, orig_path)
+        # lib/pythonX.X/site-packages
+        relto_pth = os.path.join('../../..', relto_original)
+        lines[i] = '{}\n'.format(relto_pth)
```

Comment:
I'd rather see:

```python
if val.startswith('import '):
    continue

orig_val = val
val = os.path.abspath(val, site-packages)
val = os.path.relpath(val, site-packages)
changed |= orig_val == val
```

That way you don't need to hard-code the fact that site-packages is three directories deep, which seems more foolproof to me, albeit quite unlikely to give different behavior.